package/MDAnalysis/transformations/rotate.py

```diff
@@ -76,31 +77,44 @@ def rotateby(angle, direction, center="geometry", pbc=None, ag=None, position=No
         or 'mass'
     pbc: bool or None, optional
         If True, all the atoms from the given AtomGroup will be moved to the unit cell
-        before calculating the center
-    position: list, optional
+        before calculating the center. Warning: Wrapping/unwrapping the trajectory or 
+        performing PBC corrections may not be possible after rotating the trajectory. 
+    position: array-like, optional
         list of the coordinates of the point from where a custom axis of rotation will
         be defined. 
 
     Returns
     -------
     :class:`~MDAnalysis.coordinates.base.Timestep` object
-    
+
     '''
     pbc_arg = pbc
-    if position and len(position)>2:
-        position = position
-    elif isinstance(ag, AtomGroup):
-        if center == "geometry":
-            position = ag.center_of_geometry(pbc=pbc_arg)
-        elif center == "mass":
-            position = ag.center_of_mass(pbc=pbc_arg)
-        else:
-            raise ValueError('{} is not a valid argument for center'.format(center))
+    angle = np.deg2rad(angle)
+    if position:
+        if len(position)!=3:
```

Comment:
I mentioned it already; `len` on an array is a terrible idea. An array with a (1, 3) shape will have length 3, the same goes for an array of shape (3, 3). First convert to an array with [`np.asarray`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.asarray.html), then compare the shape with `position.shape != (3, )`. You may be permissive and accept (1, 3) as a shape as well.